bugz.py

```diff
@@ -0,0 +1,205 @@
+import pandas as pd
+
+from lib.bugzilla_conn import BugzillaAPIClient
+
+from database import (
+    Database,
+    ReportBugzillaQEVerifyCount,
+    ReportBugzillaQENeeded
+)
+
+PRODUCTS = ["Fenix", "Focus", "GeckoView"]
+FIELDS = ["id", "summary", "flags", "severity",
+          "priority", "status", "resolution"]
+
+
+class Bugz:
```

Comment:
prefer a full name here consistent with other classnames.  Not sure if `Bugzilla`  or something else makes more sense here?  I see you already have `BugzillaHelper`, `BugzillaClient` and `BugzillaAPIClient`

Reply:
I can not call it Bugzilla because we are installing the python-bugzilla module which is bugzilla.py and the naming conflicts (see requirements.txt) .. I can pick a better name though if that helps